h2o-core/src/main/java/water/persist/PersistManager.java

```diff
@@ -73,6 +76,7 @@ public PersistStatsEntry() {
   }
 
   private Persist[] I;
+  private PersistHex HEX = new PersistHex(); // not part of I because it cannot be a backend for DKV
```

Comment:
typo `backed` vs `backend`

Reply:
backend is IMHO intended, PM uses this terminology

Reply:
Sorry, overlooked the rest. Makes sense now.